hack/jenkins/e2e.sh

```diff
@@ -104,8 +104,22 @@ else
 
     sudo flock -x -n /var/run/lock/gcloud-components.lock -c "gcloud components update -q" || true
 
-    GITHASH=$(gsutil cat gs://kubernetes-release/ci/latest.txt)
-    gsutil -m cp gs://kubernetes-release/ci/${GITHASH}/kubernetes.tar.gz gs://kubernetes-release/ci/${GITHASH}/kubernetes-test.tar.gz .
+    # VERSION_FILE defaults to "latest", but can also be set to "stable" for
+    # official releases.
+    VERSION_FILE="latest"
```

Comment:
Yeah, I was just about to notice that. :p 

I forgot I made a giant, flat script.

I would still make this consistent with the other pieces and make the changes I suggested, but make them VERSION_FILE="latest" and BUCKET="ci"